src/include/switch_frame.h

```diff
@@ -87,6 +87,7 @@ typedef struct switch_frame_geometry {
 	payload_map_t *pmap;
 	switch_image_t *img;
 	struct switch_frame_geometry geometry;
+	switch_time_t received_ts;
```

Comment:
We already have a timestamp on the frame. Do we really need another ?

Reply:
sure, I will look at that other timestamp to see if it is timestamped at the extraction of the frame from the packet or at the reception of the packet like the one I am using, even both should be the same most of the time.